haas_websocket/main.py

```diff
@@ -0,0 +1,89 @@
+import os
+import sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
what does this do?

Reply:
Not needed anymore, used to be useful for imports but now since it's project based it's not needed. Will remove.

Reply:
Actually, turns out this is useful. It can't perform imports without setting the working path to the path of the repo.